queries.sql

```diff
@@ -19,3 +19,11 @@ SELECT * FROM ANIMALS WHERE WEIGHT_KG >= 10.4 AND WEIGHT_KG <= 17.3;
 
 
 
+
+-- Day 2
+SELECT COUNT(NAME) AS NUMBER_OF_ANIMALS FROM ANIMALS
+SELECT COUNT(NAME) AS NUMBER_OF_ANIMALS FROM ANIMALS WHERE ESCAPE_ATTEMPTS <=0;
+SELECT AVG(WEIGHT_KG) AS AVERAGE_WEIGHT  FROM ANIMALS;
+SELECT NEUTERED ,  MAX(ESCAPE_ATTEMPTS) AS  ESCAPES  FROM ANIMALS  GROUP BY NEUTERED;
+SELECT SPECIES, MIN(WEIGHT_KG), MAX(WEIGHT_KG) FROM ANIMALS GROUP BY SPECIES;
+SELECT SPECIES, AVG(ESCAPE_ATTEMPTS) FROM ANIMALS WHERE  DATE_OF_BIRTH BETWEEN '01-01-1990' AND '01-01-2000'  GROUP BY SPECIES;
```

Comment:
- [OPTIONAL] This query is good. But what if an animal was born on the 21st March 2000? Currently, your query doesn't cover those animals. You only cover animals born on the 1st day and 1st month of 1990 and 2000.
Please update this query to cover animals born in between the given years. 

```sql
SELECT species, AVG(escape_attempts) FROM animals
   GROUP BY species
   WHERE  date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/31/12';
```